Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -78,13 +78,7 @@ def run
             casks = args.formula? ? [] : Cask::Caskroom.casks
             formulae + casks
           elsif args.named.present?
-            if args.formula?
-              args.named.to_formulae
-            elsif args.cask?
-              args.named.to_casks
-            else
-              args.named.to_formulae_and_casks
-            end
+            args.named.to_formulae_and_casks_with_taps
```

Comment:
Before this changes this was equivalent to `#to_formulae_and_casks` since the `--cask` and `--formula` options get passed to that method automatically by the parser.